Backend/ManagerLayer/AccessControl/AuthorizationManager.cs

```diff
@@ -94,6 +93,15 @@ public string ValidateAndUpdateSession(string token, Guid userId)
             return null;
         }
 
+        public Session ExistingSession(User user)
+        {
+            using (var _db = new DatabaseContext())
+            {

```

Comment:
Need to move this to dataaccess layer

Reply:
I thought we agreed to place it in the manager?